pkg/k8sutils/k8sutils.go

```diff
@@ -168,6 +168,15 @@ func GetService(svcName string, svcNS string) (*v1.Service, error) {
 
 }
 
+// DescribeService updates the service status and returns the status part
```

Comment:
lets change the comment now.. as we are not updating it
same for describe deployment and statefulset

Reply:
I'll do that separately. Eventually we may have to put in an update query before returning the status too. That the Get is getting the latest status works for us for now.